subt/docker/robotika/Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM robotika/subt-base:2020-09-22
+FROM robotika/subt-base:2020-11-11
 
 # ADD always downloads the file to calculate checksum, RUN checksum only the cmdline
-RUN curl -O# http://osgar.robotika.cz/subt/virtual/model/mdnet3.64.64.13.4.relu.pth
-RUN curl -O# http://osgar.robotika.cz/subt/virtual/model/ssd_inception_v2_201009/frozen_inference_graph.pb
-RUN curl -O# http://osgar.robotika.cz/subt/virtual/model/ssd_inception_v2_201009/cv_graph.pbtxt
+RUN curl -O# http://osgar.robotika.cz/subt/virtual/model/mdnet3.64.64.13.4.relu.pth 2>&1
+RUN curl -O# http://osgar.robotika.cz/subt/virtual/model/ssd_inception_v2_201009/frozen_inference_graph.pb 2>&1
+RUN curl -O# http://osgar.robotika.cz/subt/virtual/model/ssd_inception_v2_201009/cv_graph.pbtxt 2>&1
 
-RUN sudo apt-get update && sudo apt install -y ros-melodic-teleop-twist-keyboard python-laser-geometry
```

Comment:
I do not see the `python-laser-geometry` in base image, but maybe I am overlooking it?

Reply:
You are right! It got lost during merge/rebase on my side. Thanks for noticing!

Reply:
A new base image is being uploaded.

Reply:
sha256:e90f52a48410cb1351645c4da3e61d542a9ef785d12adb020ae1822a322fd5f0

Reply:
OK, build/download passed, thanks m.